src/lib/getDiscordVars.ts

```diff
@@ -0,0 +1,6 @@
+export function getDiscordVars() {
+  const discordWebhookEnVar = 'DISCORD_WEBHOOK',
+    discordWebhook = process.env[discordWebhookEnVar],
+    discordUsernameEnVar = 'DISCORD_USERNAME',
+    discordUsername = process.env[discordUsernameEnVar]
+}
```

Comment:
You don't return anything from this function, those variables won't be available for the caller